packages/odyssey-react-mui/src/Status.tsx

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Chip } from "./";
+
+export interface StatusProps {
+  /**
+   * Determine the color and icon of the alert
+   */
+  severity?: "default" | "error" | "info" | "success" | "warning";
+
+  /**
+   * The text content of the status
+   */
+  label: string;
```

Comment:
I put this in Jordan's PR too. If we're just passing-through props, it makes more sense to use `ChipProps["label"]` here since we don't control the type of this prop.

The problem with this approach in the long-run is if we start controlling this ourselves. Then we'd need to go back and change the prop type to our own instead.

Reply:
Hm. Theirs is `node`, but I think we want ours more limited than that for `Status`. Would `text` be more appropriate here?

Reply:
We can also `Omit` too:

```suggestion
  label: Omit<AlertProps['label'], ReactNode>`;
```

But I see what you're saying now! Yeah, if we wanna limit it to a string, you're making a lot of sense since _we_ own that prop type.

Reply:
On your comment about `text`, that works too as well as `description`. Either one.

I like making the name different from MUI when the meaning is different.